onebusaway-android/src/main/res/xml/embeddedsocial_searchable.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) Microsoft Corporation. All rights reserved.
+  ~ Licensed under the MIT License. See LICENSE in the project root for license information.
+  ~
+  -->
+
+<searchable
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        android:hint="@string/es_hint_search_topics"
+        android:label="@string/es_lib_name"
+        android:queryAfterZeroResults="true"
+        android:voiceSearchMode="showVoiceSearchButton|launchRecognizer"
+        android:searchSuggestAuthority="com.joulespersecond.seattlebusbot.embeddedsocial_searchprovider"
```

Comment:
Same here, re: build flavor - can we avoid hard-coding the first part of the authority name here so we don't have to override this file for each build flavor?